hera_pspec/plot.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import pyuvdata
+from hera_pspec import conversions
+import matplotlib.pyplot as plt
+import copy
+
+"""
+def ax():
+    fig, (ax1, ax2) = plt.subplots(1, 2, sharex=True, sharey=True)
+    do_my_stuff_with_first_axis(ax=ax1)
+    do_my_stuff_with_second_axis(ax=ax2)
+"""
+
+def delay_spectrum(uvp, blpairs, spw, pol, average_blpairs=False, 
+                   average_times=False, fold=False, plot_noise=False, 
+                   delay=True, deltasq=False, little_h=True, 
+                   legend=False, ax=None):
```

Comment:
can we add a kwarg `copy=True` which will copy the uvp object. certain operations (averaging and folding) edit the object in place, but we should be able to plot a `uvp` object without permanently changing its state. So something like
```
if copy:
    uvp = copy.deepcopy(uvp)
```
as the first line of the function

Reply:
nvm, i see that you have a `deepcopy` further down! thanks